SR2021/2021-02-14-league-2-followup.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Teams who attended the leagues and are active
+subject: Preparing for league 3
```

Comment:
I'm actually not that happy with this subject line, would be great to have some other ideas.
I wanted to avoid it seeming to focus on support, because these teams are doing fine and we don't want to encourage them too much to ask for a mentor.

Reply:
Support is basically the entire content of this email, though. If that's _not_ the focus, I don't know what is.

We could put some reminders about the next league, friendlies etc, but then we should also add to the other emails.

Reply:
@WillB97 said (on the other thread):
> Probably something more like "Need extra support?" over directly mentioning mentors in the subject.

These teams are already doing fairly well, which was why I was trying to avoid the framing being around them needing support. If we had more mentors it could be something like "Mentors now available" (framing it as news), but given that we don't have many mentors that might be misleading.

Reply:
Now that we have the friendlies in here, this subject makes a lot more sense, so I think we're good now :)

Reply:
Can we go with something like "Just checking in" or is that way too informal